openassessment/xblock/static/js/src/lms/oa_response.js

```diff
@@ -247,7 +247,7 @@ OpenAssessment.ResponseView.prototype = {
      array of strings: The current response texts.
      **/
     response: function(texts) {
-        var sel = $('.submission__answer__part__text__value', this.element);
+        var sel = $('form.response__submission .submission__answer__part__text__value', this.element);
```

Comment:
Sorry, I didn't see response__submission. If you just get rid of the form part of the selector, :+1: